setup.py

```diff
@@ -360,14 +360,16 @@ def run(self) -> None:
         for ext in self.extensions:
             if isinstance(ext, CMakeExtension):
                 print(f"Building CMake extension {ext.name}")
-                with tempfile.TemporaryDirectory() as build_dir:
```

Comment:
We should remove the tempfile import since we're not using it anymore.

Reply:
Good catch, thanks! All fixed!